BUILDING.md

```diff
@@ -0,0 +1,51 @@
+# Building RKE2
+
+## Prerequisites
+
+By default, RKE2 is built with Dapper which uses Docker. To build RKE2 you will need to install these packages:
+- bash
+- docker
+- gcc (CGO, don't ya know, if using `scripts/build` directly)
+- go (check the `go.mod` for which series of go, e.g. 1.14.x, 1.15.x, etc)
+- make
+
+When running RKE2 you will also need to install these packages:
```

Comment:
on my first read-through, i didnt catch that these two were runtime deps. caused me to think i needed them at build time and raise an eyebrow. might be worth either moving these runtime deps or giving them a more explicit sub-heading